pkg/sql/unsafesql/unsafesql_test.go

```diff
@@ -255,3 +257,35 @@ func TestAccessCheckServer(t *testing.T) {
 		})
 	}
 }
+
+// panickingStatement is a mock Statement that panics during formatting
+type panickingStatement struct{}
+
+func (ps panickingStatement) String() string {
+	return "panicking statement"
+}
+
+func (ps panickingStatement) Format(ctx *tree.FmtCtx) {
+	panic("deliberate panic for testing")
```

Comment:
trying to find an exact statement tree which panics when translating into a query string seems difficult, this is a quick way around it to test the panic catching behavior.

Reply:
reasonable :)